integrated_channels/degreed/exporters/learner_data.py

```diff
@@ -34,7 +34,9 @@ def get_learner_data_records(
         """
         # Degreed expects completion dates of the form 'yyyy-mm-dd'.
         completed_timestamp = completed_date.strftime("%F") if isinstance(completed_date, datetime) else None
-        if enterprise_enrollment.enterprise_customer_user.get_remote_id() is not None:
+        if enterprise_enrollment.enterprise_customer_user.get_remote_id(
```

Comment:
Do we need to include a test?

Reply:
totally, going to emulate what I did for sapsf shortly

Reply:
done